src/state/pages/selectors.js

```diff
@@ -95,6 +95,7 @@ export const getPageTreePages = createSelector(
       .map((pageCode) => {
         const isEmpty = !(pageChildren[pageCode] && pageChildren[pageCode].length);
         let hasPublishedChildren = false;
+        const parentStatus = pages[pageCode].parentCode && pages[pages[pageCode].parentCode].status;
```

Comment:
Are you sure `pages[pageCode]` and `pages[pages[pageCode].parentCode]` can't be undefined here? Wouldn't it be better to use a more forgiving approach, like using lodash `get` and checking if the result is defined?